packages/augur-ui/src/modules/account/components/open-markets.tsx

```diff
@@ -25,12 +26,15 @@ export default class OpenMarkets extends Component<OpenMarketsProps> {
     this.renderRows = this.renderRows.bind(this);
   }
 
-  renderRows(market: any) {
+  renderRows(market: Partial<MarketData>) {
     const { marketsObj } = this.props;
-
+    const positionValueChange =
+      marketsObj[market.id] &&
+      marketsObj[market.id].myPositionsSummary &&
+      marketsObj[market.id].myPositionsSummary.valueChange  || formatNumber(0);
```

Comment:
Nitpick: there are two spaces instead of 1 here
`marketsObj[market.id].myPositionsSummary.valueChange  ||`

Reply:
updated PR